### DISCUSSION TITLE
DateTime field step attribute

### DISCUSSION THREAD

**Discussion by fabricecw** [laravel/nova-issues#4137] [Ideas & Feature Requests]
- Laravel Version: v8.83.10
- Nova Version: 4.2.7
- PHP Version: PHP 7.4
- Browser type and version: Chrome 100.0

### Description:
If you use the `DateTime` field, the `step` attribute is hardcoded to 1 in the Nova component. This means the user always needs to input seconds, even if you only need hours and minutes. 

Other fields support the `step` method and it could be implemented the exact same way. For example with a `step` value of 60, you can disable / hide the seconds input in the picker.

### Detailed steps to reproduce the issue on a fresh Nova installation:
1. Fresh Nova instalaltion
2. Migrate a table with a `dateTime` column.
3. Create a resource with Nova's `DateTime` field.
4. Use the datetime picker

**Comment by crynobone** [comment]
Implemented: https://nova.laravel.com/docs/4.0/resources/date-fields.html#options
